README.md

```diff
@@ -29,6 +29,13 @@ Starts and stops a metaprob-viz server instance listening on
 port 8081. This server manages HTTP and websocket traffic from the
 client renderer.
 
+`(start-server!)` is variadic, and may be called with arguments
+specifying the port, or IP and port, to listen on:
+
+    (start-server!)                      ;; defualt, start on 127.0.0.1:8081
+    (start-server! 9999)                 ;; start on 127.0.0.1:9999
+    (start-server! "192.168.1.100" 8088) ;; start on 192.168.1.100:8088
```

Comment:
My feeling is that positional arguments should maintain position between different arity overloads. 🙂

Reply:
Yes, I tend to agree, and initially had this working that way. But it seemed weird to use

    (start-server! 8088 "192.168.1.100")

since generally when talking about IP/port combinations, the IP naturally comes first. Not sure what's right, here.

Reply:
Optional configuration map?